src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -133,6 +133,50 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
   }
 }
 
+/**
+ * Uses Drupal content via a new GraphQL query or the cached result of a
+ * previous query. This is where the cache is saved.
+ *
+ * @param {Object} buildOptions
+ * @return {Object} - The result of the GraphQL query
+ */
+async function getSideNavsViaGraphQL(buildOptions = global.buildOptions) {
+  const { getOptions } = require('../options');
```

Comment:
I just noticed this; does this need to be a dynamic `require`? Could tihs be put at the top of the module?

Reply:
I tried it and this was the only way I could get it working.

Reply:
By that, do you mean the build would fail?

Reply:
When I tried to view a page using the preview server, it would throw an error.